Xamarin.Forms.Platform.Android/Extensions/FormsInitExtensions.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Xamarin.Forms
+{
+	public static class FormsInitExtensions
+	{
+		public static IFormsInit WithFlags(this IFormsInit init, params string[] flags)
+		{
+			init.PreInit(() => Forms.SetFlags(flags));
```

Comment:
what happens in Forms if I call

```csharp
Forms.SetFlags("some_flag_one");
Forms.SetFlags("some_flag_two");
```

does `flag_two` override `flag_one` or is it appended?
The reason I ask is because I wonder if a user might

```csharp
Forms.Create(this, bundle)
    .WithFlags("some_flag_one")
    .WithFlags("some_flag_two")
    .Init()
```

Is it idempotent?

If not, I think you'd have to re-think the extension. Should you assert/throw if they call it more than once, or should you build up a list of flags and delay the invoke?

Reply:
I will make it as idempotent, thanks for idea

Reply:
done